pagopa/io-app
# feat(Piattaforma Notifiche): [IAMVL-89] Add a CTA for viewing a notification in the PN website

## Short description
### ⚠️ **This PR depends on #4083**
This PR adds a CTA for taking the user to the PN website. The URL is taken from `io-services-metadata` and, if not available, the CTA is not presented.

## List of changes proposed in this pull request
| Available | N/a |
| - | - |
| ![Simulator Screen Shot - iPhone 13 - 2022-07-27 at 15 39 49](https://user-images.githubusercontent.com/467098/181267821-346f1427-8c53-48c7-9c89-8e4df8e7ec11.png) | ![Simulator Screen Shot - iPhone 13 - 2022-07-27 at 15 50 58](https://user-images.githubusercontent.com/467098/181267884-82d3cd5d-ce27-49c4-884f-156800915a5d.png) |

https://user-images.githubusercontent.com/467098/181267947-003f49e0-8759-4407-bb5e-4a925f11efdc.mp4

## How to test
Use the [IIP-18-add-pn-endpoints](https://github.com/pagopa/io-dev-api-server/tree/IIP-18-add-pn-endpoints) branch of the dev server and add this to your `config.json`:
```
{
  "messages": {
    "pnCount": 1
  },
  "services": {
    "includePn": true
  }
}
```
Make sure you set `PN_ENABLED=YES` to `.env` and run the app. By changing `config.pn` in `/src/payloads/backend.ts` of the dev server you can simulate each scenario.
